Source/AliveLibAO/Abe.cpp

```diff
@@ -7550,8 +7550,8 @@ void Abe::State_77_WellBegin_430F10()
     if (field_10_anim.field_4_flags.Get(AnimFlags::eBit18_IsLastFrame))
     {
         field_114_gnFrame = 15;
-        SFX_Play_43AD70(25u, 0, this);
-        field_FC_current_motion = eAbeStates::State_78_InsideWellLocal_4310A0;
+        SFX_Play_43AD70(SoundEffect::WellEnter_25, 0, this);
+        field_FC_current_motion = static_cast<eAbeStates>((static_cast<int>(field_FC_current_motion) + 1));
```

Comment:
cast is redundant because it isn't strongly type, but eventually we want this to be strongly typed so I think you should do something like the other states do:

```
switch(field_FC_current_motion)
{
case state_77_WellBegin_430F10:
 field_FC_current_motion = State_78;
 break;

// Whatever at least the other case is

default:
LOG_ERROR(field_FC_current_motion );
ALIVE_FATAL("Unhandled next well state);
break;
}

```